### ISSUE TITLE
missing dependency  vue-runtime-helpers

### ISSUE THREAD

**Issue by mariusa** [KitchenStories/vue-gallery-slideshow#33]
```
This dependency was not found:

* vue-runtime-helpers in ./node_modules/vue-gallery-slideshow/dist/js/vue-gallery-slideshow.js

To install it, you can run: npm install --save vue-runtime-helpers

```

**Comment by waltergammarota** [comment]
I had the same issue, but running
`yarn add vue-runtime-helpers `
solve it, probabbly it will work with npm too

**Comment by mariusa** [comment]
It should be added to dependencies in this project's package.json, like all other libraries do

**Comment by headione** [comment]
@mariusa @waltergammarota Thanks for this hint! I've added it now to the project and released a patch (v1.5.1) the fixes that issue.

**Comment by mariusa** [comment]
Thanks! Please publish to npm too:
https://www.npmjs.com/package/vue-gallery-slideshow
